openghg/standardise/_standardise.py

```diff
@@ -376,7 +376,7 @@ def standardise_bc(
 
 
 def standardise_footprint(
-    filepath: str | Path | list,
+    filepath: multiPathType,
```

Comment:
I prefer the more explicit option here, but it's not really a big deal.
```suggestion
    filepath: str | Path | list[str | Path],
```

Reply:
For what I'm trying to do at the moment I could do with starting to make `Footprint` and `ObsSurface` be consistent in terms of the `filepath` type - however, we can see if this makes more sense as the more verbose type once that's ironed out a bit hopefully.